server/template.js

```diff
@@ -110,8 +136,14 @@ export default ({ url, bodyClass, title, description }) => marinate`
   <!-- Polyfills -->
   <script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=fetch,Object.entries,Array.from,Array.prototype.includes,String.prototype.startsWith,String.prototype.includes,Map,Set,requestAnimationFrame,Array.prototype.@@iterator"></script>
 
+<<<<<<< HEAD
+  <script src="${manifest['vendor.js']}"></script>
+  <script src="${manifest['app.js']}"></script>
+</body>
+=======
```

Comment:
This isn't required as `scriptsHTML` takes care of this.

Reply:
I removed these lines, is there any other change required?

Reply:
<<<<< HEAD is part of git conflict. We need to resolve that. And the resolution is to remove the script part as `scriptHTML` is gonna put them.